content/applications/general/email_communication/email_domain.rst

```diff
@@ -83,81 +83,95 @@ and has not been altered during transport.
+- `NameCheap <https://www.namecheap.com/support/knowledgebase/article.aspx/317/2237/how-do-i-add-
+  txtspfdkimdmarc-records-for-my-domain/>`_
 - `CloudFlare DNS <https://support.cloudflare.com/hc/en-us/articles/360019093151>`_
 - `Google Domains <https://support.google.com/domains/answer/3290350?hl=en>`_
 - `Azure DNS <https://docs.microsoft.com/en-us/azure/dns/dns-getstarted-portal>`_
 
-To fully test your configuration, the tool `Mail-Tester <https://www.mail-tester.com/>`_ will give 
-you a full overview of the content and configuration you have in one email sent! Mail-Tester can 
+To fully test your configuration, the tool `Mail-Tester <https://www.mail-tester.com/>`_ will give
+you a full overview of the content and configuration you have in one email sent! Mail-Tester can
 also be used for other lesser known providers.
 
 Use a default email address
 ===========================
 
-To force the email address from which emails are sent, you need to create the following key in
-the System Parameters of the database:
+To force the email address from which emails are sent, a combination of the following keys needs to
+be set in the system parameters of the database:
+
+- `mail.default.from`: accepts the local part or a complete email address as value
+- `mail.default.from_filter`: accepts a domain name or a full email address as value
+
+.. note::
+   The `mail.default.from_filter` works only for `odoo-bin` configurations, otherwise this
+   parameter can be set using the `from_filter` field on `ir.mail_server`.
+
+If the email address of the author does not match `mail.default.from_filter`, the email address is
+replaced by `mail.default.from` (if it contains a full email address) or a combination of
+`mail.default.from` and `mail.catchall.domain`.
 
-- If ``mail.default.from`` is set, and contains a full email address, all outgoing emails are sent
-  from the given address. This is a requirement to use `Outlook with Odoo 
-  <https://docs.microsoft.com/en-us/exchange/mail-flow-best-practices/how-to-set-up-a-multifunction-device-or-application-to-send-email-using-microsoft-365-or-office-365#option-1-authenticate-your-device-or-application-directly-with-a-microsoft-365-or-office-365-mailbox-and-send-mail-using-smtp-auth-client-submission>`_.
+If the `from_filter` contains a full email address, and if the `mail.default.from` is the same as
+this address, then all of the email addresses that are different from `mail.default.from` will be
+encapsulated in `mail.default.from`.
 
-You access the **System Parameters** in :ref:`developer mode <developer-mode>` in the :menuselection:`Settings -->
-Technical --> Parameters --> System Parameters` menu.
\ No newline at end of file
+You access the **System Parameters** in :ref:`developer mode <developer-mode>` in the
+:menuselection:`Settings --> Technical --> Parameters --> System Parameters` menu.
```

Comment:
- Delete `You`
- Capitalize the first letter of `access`
- Replace the bold formatting around `System Parameters`  with a `:guilabel:`
- Replace `in` with `by activating`
- Replace `in the` with `and going to`
- I also feel like this sentence `Access the...menu.` should be moved to the beginning of the section, before `To force the email address...of the database.`